README.md

```diff
@@ -26,6 +26,15 @@ $ pip3 install "odoo-analyse[graph]"
 
 `-s /path/to/data.json` .. Store the loaded modules in a file
 
+Or if you want to output it to `stdout`:
+
+`-s -` .. Outputs the loaded modules to stdout
+
+### Loading files
+`-l /path/to/data.json` .. Loads the JSON data from a file which you can use for detailed analysis/graphs.
+This serves as a cache-like system to analyse repositories or Odoo installations by first saving the data into a file and then reusing it.
```

Comment:
This section is already there. Only a note about the usage of `-` to load the cache from stdin would be cool.